src/events/guildMemberRemove.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  EmbedBuilder, type Client, type GuildMember,
+  type TextChannel, type ColorResolvable
+} from "discord.js";
+import { genColor, genRGBColor } from "../utils/colorGen.js";
+import Vibrant from "node-vibrant";
+import sharp from "sharp";
+
+export default {
+  name: "guildMemberRemove",
+  event: class GuildMemberRemove {
+    client: Client;
+
+    constructor(client: Client) {
+      this.client = client;
+    }
+
+    async run(member: GuildMember) {
+      const id = "1079612083307548704";
+      const channel = await member.guild.channels.cache.find(channel => channel.id === id).fetch() as TextChannel;
+      const avatarURL = member.displayAvatarURL();
+
+      const embed = new EmbedBuilder()
+        .setAuthor({
+          name: `•  ${member.nickname == null ? member.user.username : member.nickname}`,
+          iconURL: avatarURL
+        })
+        .setTitle("Goodbye!")
+        .setDescription("idk what to say here yet")
```

Comment:
Uhm... pretty straight-forward, probably something like "@(username) has left the server"? I will make a suggestion for that, just in case you're going to add it to your batch. Saves us time 👍 
```suggestion
        .setDescription(`**@${member.user.username}** has left the server 😥`)
```
For now just some generic message, but ofcourse people can change this one in their Dasshubosudo 💯 ;)

Reply:
next commit has this implemented